CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.0 (2019-07-22)
```

Comment:
Do we need to update the date since we didn't merge this today?

Reply:
Done